src/core/services/compiler/compiler.js

```diff
@@ -10,74 +10,91 @@ angular
 
 /**
  * @ngdoc service
- * @name $mdCompiler
+ * @name $mdCompilerProvider
  * @module material.core.compiler
  * @description
- * The $mdCompiler service is an abstraction of AngularJS's compiler, that allows developers
- * to easily compile an element with options like in a Directive Definition Object.
+ * The `$mdCompiler` is able to respect the AngularJS `$compileProvider.preAssignBindingsEnabled` state when using
+ * AngularJS versions between 1.5.10 and 1.7.0.
```

Comment:
It sounds like it respects it for 1.7.0 as well. How about:
```js
 * The `$mdCompiler` is able to respect the AngularJS `$compileProvider.preAssignBindingsEnabled` state when using
 * AngularJS versions from 1.5.10 until (but not including) 1.7.0.
```
?

Reply:
`$compileProvider.preAssignBindingsEnabled` is removed from AngularJS in 1.7.0. Maybe that's your point? I'll try to refine it.